app/server/appsmith-plugins/openAiPlugin/src/main/resources/editor/vision.json

```
@@ -0,0 +1,113 @@
+{
+  "identifier": "VISION",
+  "controlType": "SECTION",
+  "conditionals": {
+    "show": "{{actionConfiguration.formData.command.data === 'VISION'}}"
+  },
+  "children": [
+    {
+      "label": "Models",
+      "tooltipText": "Select the model for response generation",
+      "subtitle": "ID of the model to use.",
+      "isRequired": true,
+      "propertyName": "vision_model_id",
+      "configProperty": "actionConfiguration.formData.visionModel.data",
+      "controlType": "DROP_DOWN",
+      "initialValue": "",
+      "options": ["gpt-4-vision-preview"],
```

```suggestion
      "options": [],
```
Options will be fetched from the api

---

Open AI is going to discontinue `gpt-4-vision-preview` model soon and will publish something line gpt-4-vision. So it is always best to fetch the values from the api.